src/main/java/racingcar/MovingStrategy.kt

```diff
@@ -0,0 +1,5 @@
+package racingcar
+
+interface MovingStrategy {
```

Comment:
```suggestion
@FunctionalInterface
interface MovingStrategy {
    fun movable(): Boolean
}
```
@FunctionalInterface를 붙여서 함수형 인터페이스임을 알려주면 좋을 것 같아요!